randovania/db_integrity/integrity.py

```diff
@@ -0,0 +1,67 @@
+from collections.abc import Iterator
```

Comment:
Does this file need to be in a new module?

Reply:
probably not, didnt know where else to put it.
logic_database sounds wrong cause thats not a code file
 layout?

Reply:
game_description? There's literally `integrity_check.py` there.

Reply:
and also create game_description for within `games` then too?

Reply:
does it need to be a new folder?

Reply:
IDK putting it right into games felt kinda weird to me, but I guess that worsk.